lib/piggly/parser/grammar.tt

```diff
@@ -359,7 +359,7 @@ grammar Piggly
   end
 
   rule dollarQuoteMarker
-    '$' tag:( [a-z\200-\377_] [a-z\200-\377_0-9]* )? '$' <Piggly::Parser::Nodes::TDollarQuoteMarker>
+    '$' tag:( [a-z\w_] [a-z\w_0-9]* )? '$' <Piggly::Parser::Nodes::TDollarQuoteMarker>
```

Comment:
If Unicode is not allowed, this `a-z\w_0-9` could probably be simplified to `\w`, but the first part should _not_ match numbers, so probably `[a-z][\w]*` is the right pattern (must start with a letter).